t/commands.t

```diff
@@ -69,6 +69,13 @@ subtest "service command" => sub {
     # Clear it for next test.
     undef @added_services_modules;
 
+    # Running services under the same namespace
+    wait_for_future( $command->service('Ta::*')->get->{code}->() )->get;
+    cmp_deeply(\@added_services_modules, ['Ta::Sibling1', 'Ta::Sibling2'], 'Added modules under the namespace');
+    # Clear it for next test.
+    undef @added_services_modules;
```

Comment:
I'd recommend "set to empty list" instead, `undef` on arrays is a bit unintuitive (looks more like "set all entries to undef", same way `chomp` works):

```suggestion
    @added_services_modules = ();
```